css/styles.css

```diff
@@ -0,0 +1,337 @@
+body {
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    background-color: #0D0D11;
+}
+
+h1,
```

Comment:
Avoid styling tags directly
except for body, ul, a, img.

**Bad:**

section {}
header {}

**Good:**

body {}
ul {}
a {}
img {}
.page-header {}
.block-section {}